utils/remote-run

```diff
@@ -136,8 +137,8 @@ class LocalCommandRunner(CommandRunner):
         return ['/usr/bin/sftp', '-b', '-', '-q', '-D', self.sftp_server_path]
 
 def find_transfers(args, source_prefix, dest_prefix):
-    if source_prefix.endswith(os.path.sep):
-        source_prefix = source_prefix[:-len(os.path.sep)]
+    if source_prefix.endswith(posixpath.sep):
+        source_prefix = source_prefix[:-len(posixpath.sep)]
```

Comment:
Arguably these *should* be using host-style paths, but I guess the tests aren't going to do that any time soon.